tests/pos/i5386.scala

```diff
@@ -22,4 +23,12 @@ object Test {
   { println("1"); 1 } + { println("2"); 2}
 
   !(try true finally{()})
+
+
+  class C {
+    def foo: 1 = 1
```

Comment:
To make sure this works as expected, the body of this method should contain a print, and this test should be a run test with a check file with that print.

Reply:
OK, but somebody else should do that, please. I was not volunteering to take this on issue, but since it was blocked otherwise, had to step in. So I won't spend any more context switches on this.